Parser/tokenizer.h

```diff
@@ -33,6 +33,35 @@ struct token {
     const char *start, *end;
 };
 
+enum tokenizer_mode_kind_t {
+    TOK_REGULAR_MODE,
+    TOK_FSTRING_MODE,
+};
+
+#define MAX_EXPR_NEXTING 2
+
+typedef struct _tokenizer_mode {
+    enum tokenizer_mode_kind_t kind;
+
+    // TODO: we probably can infer this without storing it
+    // from the other information available here.
+    int format_spec;
+
+    int bracket_stack;
+    int bracket_mark[MAX_EXPR_NEXTING];
+    int bracket_mark_index;
+
+    char f_string_quote;
+    int f_string_quote_size;
+    int f_string_raw;
+    const char* f_string_start;
+    const char* f_string_multi_line_start;
+
+    int last_expr_size;
+    int last_expr_end;
```

Comment:
```suggestion
    Py_ssize_t last_expr_size;
    Py_ssize_t last_expr_end;
```
CI/CD says about conversion size_t to int (and possible  loss of data)
I think, we can safely change types of this fields from `int` to `Py_ssize_t`.
If so, the warnings will go away.